src/raygun.breadcrumbs.js

```diff
@@ -0,0 +1,296 @@
+/*
+ * raygun4js
+ * https://github.com/MindscapeHQ/raygun4js
+ *
+ * Copyright (c) 2017 MindscapeHQ
+ * Licensed under the MIT license.
+ */
+/* globals console */
+
+var raygunBreadcrumbsFactory = function(window, $, Raygun) {
```

Comment:
I've just copied the convention in the other files, the rum file does the same thing and we would only ever make a single rum instance as well?